libcontainer/SPEC.md

```diff
@@ -367,7 +367,31 @@ profile <profile_name> flags=(attach_disconnected,mediate_deleted) {
 }
 ```
 
-*TODO: seccomp work is being done to find a good default config*
+**seccomp** can be used to apply filters to the system calls used in
```

Comment:
Here you broke the link too.

What @cyphar showed with the `[ ]` is a markdown syntax to put links. You removed it, but we want it like that for the links :)